bundles/org.openhab.binding.zwavejs/src/main/java/org/openhab/binding/zwavejs/internal/api/dto/DeviceConfig.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.zwavejs.internal.api.dto;
+
+import java.util.ArrayList;
+
+/**
+ * @author Leo Siepel - Initial contribution
+ */
+public class DeviceConfig {
+    public String filename;
+    public boolean isEmbedded;
+    public String manufacturer;
+    public int manufacturerId;
+    public String label;
+    public String description;
+    public ArrayList<Device> devices;
```

Comment:
Just a question: is it important to use ArrayList and not just List?

Reply:
Not important, changed to List